.github/workflows/ci_test-full.yml

```
@@ -17,13 +17,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-18.04, windows-2019, macOS-10.15]
+        os: [ubuntu-latest, windows-2019, macOS-10.15]
```

I would be rather explicit or at least add a link to GH docs where is the mapping - at latest really is
there could be some confusion as actually, latest is 21 but for gh actions, it is 20
```suggestion
        os: [ubuntu-20.04, windows-2019, macOS-10.15]
```

---

Totally makes sense :]